include/oneapi/dpl/internal/scan_by_segment_impl.h

```diff
@@ -366,10 +363,11 @@ struct __sycl_scan_by_segment_impl
     }
 };
 
-template <typename _BackendTag, typename Policy, typename InputIterator1, typename InputIterator2,
-          typename OutputIterator, typename T, typename BinaryPredicate, typename Operator, typename Inclusive>
-OutputIterator
-__scan_by_segment_impl_common(__internal::__hetero_tag<_BackendTag>, Policy&& policy, InputIterator1 first1,
+// TODO required to move into oneapi::dpl::__par_backend_hetero namesace
```

Comment:
@mmichel11 Does this actually belong in `__par_backend_hetero`? I'm not so sure.
We may just want / need  to rename `__sycl_scan_by_segment_impl` to be 
`__parallel_scan_by_segment` or something similar.  Move that to `oneapi::dpl::__par_backend_hetero`, and add a fully qualified name here in this function.  
This would also mean that the related tags would stay as is (not include the changes in this PR).

Reply:
Please see #2201. If / when we merge that we can adjust this code accordingly.

Reply:
TODO has been removed

Reply:
> Please see #2201. If / when we merge that we can adjust this code accordingly.

I think at first you will merge https://github.com/uxlfoundation/oneDPL/pull/2201 into main branch. then I will merge main into my PR.